docs/patterns/observability/logs.md

```diff
@@ -57,3 +57,24 @@ Once logs have been sent to Log Analytics, that can also be consumed by Applicat
 
 You can now interact with your logs in either of these instances.
 
+
+### Configuring Outbound Proxy Support for Fluent Bit
+If you require outbound proxy support:
```

Comment:
Could you add a description around why someone would want or require outbound proxy support for fluent bit?

Reply:
@ritazh I've added a line on this, could you take a look?